statistics/src/main/java/com/duckduckgo/app/statistics/pixels/Pixel.kt

```diff
@@ -92,6 +92,7 @@ interface Pixel {
         const val DAX_SERP_CTA = "s"
         const val DAX_NETWORK_CTA_1 = "n"
         const val DAX_TRACKERS_BLOCKED_CTA = "t"
+        const val DAX_TRACKERS_BLOCKED_EXPERIMENT_CTA = "te"
```

Comment:
I don't think you need this. You should be able to use `t` and know which version we are showing by the variant. The issue with having two is that the onboarding never ends (unless we change a few things) and after the experiment is gone, users will start to see `t` again.

Reply:
You are right, I'll remove it and leave `t`